template/page-community.html.jinja

```diff
@@ -82,7 +73,174 @@
   {% endfor %}
   </div>
 </section>
-<section class="container-new py-26">
+
+
+<!-- Contribute -->
+<section class="container-new" id="run-a-network-node">
+  <div class="card-grid card-grid-2xN">
+    <div class="col d-none d-lg-block align-self-center">
+      <div class="mt-10" id="networkNode"></div>
+    </div>
+    
+    <div class="col pt-lg-5">
+      <div class="d-flex flex-column-reverse mb-8 pl-0">
+        <h2 class="h4 h2-sm">{% trans %}Run an XRP Ledger network node{% endtrans %}</h2>
+        <h6 class="eyebrow mb-3">{% trans %}Contribute to Consensus{% endtrans %}</h6>
+      </div>
+
+      <div class="col d-lg-none d-block">
+        <div class="mt-10" id="networkNode-small"></div>
+      </div>
+
+      <div class="pt-2 pt-lg-5 card-grid card-grid-2xN text-cards">
+        <div class="text-card">
+          <a class="btn-arrow" href="/the-rippled-server.html">About the Server</a>
+          <p class="mt-3 mb-0">rippled is the core peer-to-peer server that manages the XRP Ledger.</p>
+        </div>
+    
+        <div class="text-card">
+          <a class="btn-arrow" href="/run-rippled-as-a-validator.html">Become a Validator </a>
+          <p class="mt-3 mb-0">Join the Unique Node List (UNL) to contribute to consensus on XRPL.</p>
+        </div>
+      
+        <div class="text-card">
+          <a class="btn-arrow" href="/install-rippled.html">Install & Configure</a>
+          <p class="mt-3 mb-0">Install, run, and update the rippled server.</p>
+        </div>
+      
+        <div class="text-card">
+          <a class="btn-arrow" href="/troubleshoot-the-rippled-server.html">Troubleshooting</a>
+          <p class="mt-3 mb-0">Troubleshoot any issues you’re having with the rippled server.</p>
+        </div>
+      </div>
```

Comment:
This section should really pull the contents from the pages array, for these reasons:

- If the name or description of the page changes, it should automatically update here.
- In the Japanese translation, it should (automatically) get the translated page name and description, if available
- Using absolute links like this will drop people out of the translated version if they click them. Instead they should use the `{{target.prefix}}{{page.html}}` links.

Here's how you could do that:

```suggestion
      <div class="pt-2 pt-lg-5 card-grid card-grid-2xN text-cards">
        {% set curated_links = [
          "the-rippled-server.html",
          "run-rippled-as-a-validator.html",
          "install-rippled.html",
          "troubleshoot-the-rippled-server.html"
        ] %}
        {% for cc in curated_links %}
        {% set page = pages|selectattr("html", "defined_and_equalto", cc)|first %}
        <div class="text-card">
          <a class="btn-arrow" href="{{currentpage.prefix}}{{page.html}}">{% if page.top_nav_name is defined %}{{page.top_nav_name}}{% else %}{{page.name}}{% endif %}</a>
          <p class="mt-3 mb-0">{{page.blurb}}</p>
        </div>
        {% endfor %}
      </div>
```

There's one difference, which is that `top_nav_name` for `run-rippled-as-a-validator.html` is currently defined as **Join UNL** (that's what it's linked as in the top nav). IMO the link text should be consistent across both places so it's a question of whether we want to change the design or the dropdown. If we _really_ want them to be different we can code up a special case for this.

Reply:
I agree they should match. I used your idea for looping through the pages